src/main/java/org/folio/circulation/resources/LoanCollectionResourceHelper.java

```diff
@@ -0,0 +1,97 @@
+package org.folio.circulation.resources;
+
+import static org.folio.circulation.domain.representations.LoanProperties.ITEM_ID;
+import static org.folio.circulation.support.Result.of;
+import static org.folio.circulation.support.ValidationErrorFailure.singleValidationError;
+
+import org.folio.circulation.domain.Item;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.domain.RequestQueue;
+import org.folio.circulation.domain.ServicePointRepository;
+import org.folio.circulation.domain.User;
+import org.folio.circulation.domain.validation.ItemNotFoundValidator;
+import org.folio.circulation.support.Result;
+
+import java.util.concurrent.CompletableFuture;
+
+public class LoanCollectionResourceHelper {
```

Comment:
Could this be given a more specific name than helper?

If it is really a few different areas of responsibility, please could it be split up.